src/infinite-scroller.ts

```diff
@@ -85,6 +85,9 @@ export class InfiniteScroller
   /** @inheritdoc */
   @property({ type: Number }) itemCount = 0;
 
+  /** @inheritdoc */
+  @property({ type: Number }) predictedFullItemCount?: number;
```

Comment:
Should this be being used somewhere? Also it doesn't seem like it's inherited from the interface.

Reply:
Oh, good eyes! Yes, this was leftover from an earlier approach and no longer used -- removed with latest comment.